shared/src/androidUnitTest/kotlin/com/intive/picover/shared/Test.android.kt

```diff
@@ -2,13 +2,20 @@
 
 package com.intive.picover.shared
 
+import com.intive.picover.shared.koin.platformModule
 import kotlin.test.DefaultAsserter.assertTrue
 import org.junit.jupiter.api.Test
+import org.koin.core.context.startKoin
+import org.koin.test.KoinTest
 
-class AndroidGreetingTest {
+class AndroidGreetingTest : KoinTest {
```

Comment:
Next step is to adjust iOS shared unit tests - disabled for now. I have to manage running them from iOS app to table linking with Firebase dependencies